packages/coil-extension/static/acceptMonetizationRules.json

```diff
@@ -5,7 +5,7 @@
     "action": {
       "type": "modifyHeaders",
       "requestHeaders": [
-        { "header": "accept-payment", "operation": "set", "value": "webmon/*;q=0.8" }
+        { "header": "accept-monetization", "operation": "set", "value": "webmon/*;q=0.8" }
```

Comment:
It's been suggested that "monetization" is a better name, so updating to align with other documents

Reply:
nice